resources/fixtures/Lumeri/Lumeri-Eco-COB-15.qxf

```diff
@@ -21,7 +21,7 @@
   <Capability Min="58" Max="64" Preset="ColorMacro" Res1="#00ff00">Green</Capability>
   <Capability Min="65" Max="71" Preset="ColorMacro" Res1="#0000ff">Blue</Capability>
   <Capability Min="72" Max="78" Preset="ColorMacro" Res1="#ffff00">Yellow</Capability>
-  <Capability Min="79" Max="85" Preset="ColorMacro" Res1="#ff00ff">Purple</Capability>
+  <Capability Min="79" Max="85" Preset="ColorMacro" Res1="#a020f0">Purple</Capability>
```

Comment:
Why not Magenta here?

Reply:
I have no access to the manual (cannot find a trace of it in Google) and on the manufacturer pages (which probably translates Purple as the author wrote it here?).
I'm changing it to Magenta - this makes more sense, all other color channels are using 0x00 or 0xff values ans should be correct.